### DISCUSSION TITLE
Are these channels legal

### DISCUSSION THREAD

**Discussion by vdej8** [iptv-org/community#416] [Q&A] [answered]
I want to use this as a substitute for cable, and I was wonder if this is legal.

**Comment by vijaibishnoi** [comment]
[No, These Channels are not legal.](https://www.techstag.in/2022/07/iptv-m3u8-playlist-links.html)

**Comment by freearhey** [comment, marked as answer]
https://github.com/orgs/iptv-org/discussions/294
